requirements_dev.txt

```diff
@@ -235,7 +238,7 @@ ruamel-yaml-clib==0.2.7
     # via
     #   -r ./requirements.txt
     #   ruamel-yaml
-semgrep==0.122.0
+semgrep==1.0.0
```

Comment:
As it appears semgrep recently had it's `1.0` release. Maybe you can check update the `attrs` requirements now as we have fixed it. If we can update it we can finally get rid of our custom `min_len_validator`.

Reply:
not in this PR, but soon we will try it